plugin/completion.py

```diff
@@ -35,11 +35,11 @@ def format_documentation(self, content: str) -> str:
         return minihtml(self.view, content, allowed_formats=FORMAT_STRING | FORMAT_MARKUP_CONTENT)
 
     def get_content(self, documentation: str, detail: str) -> str:
-        content = ""
+        content = mdpopups.format_frontmatter({"allow_code_wrap": True})
```

Comment:
Don't we need all of the other frontmatter configuration here, too? (escapeall and magiclink?)

Reply:
I can add that.

Reply:
Well, the code here uses `minihtml` function which adds frontmatter automatically. Though it's added multiple times (twice) and not at the root of the content which seems a bit weird...

Reply:
No, that's fine as `minihtml` returns processed HTML already.
So I think we just don't need frontmatter here as the content will be already "wrappable".